client/a8c-for-agencies/components/page-section-columns/index.tsx

```diff
@@ -0,0 +1,56 @@
+import { useBreakpoint } from '@automattic/viewport-react';
+import clsx from 'clsx';
+import { ReactNode } from 'react';
+
+import './style.scss';
+
+type SectionBackground = {
+	image?: string;
+	color?: string;
+	isDarkBackground?: boolean;
+};
+
+interface ColumnProps {
+	children?: ReactNode;
+	alignCenter?: boolean;
+}
+
+const Column = ( { children, alignCenter }: ColumnProps ) => (
+	<div className={ clsx( 'page-section-column', { 'is-align-center': alignCenter } ) }>
+		<div className="page-section-column__content">{ children }</div>
+	</div>
+);
+
+interface PageSectionColumnsProps {
+	heading?: ReactNode;
+	children: ReactNode;
+	background?: SectionBackground;
+}
+
+const PageSectionColumns = ( { heading, children, background }: PageSectionColumnsProps ) => {
```

Comment:
Is there a reason why we can't reuse the `PageSection` component?

Reply:
There are some differences design-wise: heading size etc. Plus we are using Core styles here.